pandas/core/frame.py

```diff
@@ -5401,6 +5401,94 @@ def _construct_result(self, result) -> "DataFrame":
         out.columns = self.columns
         return out
 
+    @Appender(
+        """
+Returns
+-------
+DataFrame
+    DataFrame that shows the differences stacked side by side.
```

Comment:
Probably worth stating a bit about what the index and columns will look like (multiindex or not, etc.)

Reply:
added